src/wallet/components/walletSvg.tsx

```diff
@@ -0,0 +1,18 @@
+export const walletSvg = (
```

Comment:
I guess we could keep a custom component just for this special case, but ideally we still need one to create new links for the dropdown.